rpcs3/Emu/Cell/SPURecompiler.cpp

```diff
@@ -6370,6 +6370,54 @@ class spu_llvm_recompiler : public spu_recompiler_base, public cpu_translator
 		call("spu_write_channel", &exec_wrch, m_thread, m_ir->getInt32(op.ra), val.value);
 	}
 
+	template <typename T, typename U>
```

Comment:
Why moving it? Wasn't supposed to be necessary.

Reply:
Oh I understand, these are member functions.